api/app/signals/apps/signals/migrations/0069_merge_20190731_1526.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.1.10 on 2019-07-31 13:26
```

Comment:
Git merge master, then we can have only 1 new migration --- and leave this one out.

Reply:
Will fix that when presenting the final PR

Reply:
ok - will leave the conversation open so that we do not forget